README.md

```diff
@@ -4,11 +4,76 @@ When you have multiple Pipeline jobs, you often want to share some parts of the
 scripts between them to keep Pipeline scripts [DRY](http://en.wikipedia.org/wiki/Don't_repeat_yourself).
 A very common use case is that you have many projects that are built in the similar way.
 
-This plugin adds that functionality by allowing you to create “shared library script” SCM repositories.
```

Comment:
This conversion of two items in a single sentence into a list of two items is a very nice change.  Thanks for doing that.

It would have been much easier for me to read if the other white space changes had not been made in the same area.

Is there a compelling reason to change from sentence per line to a paragraph that is folded at 72 characters or 80 characters length?

Reply:
No, there is no reason to commit, would you like me to have it on a single line, because I would be more than happy to make it more readable :+1:

Reply:
I'm a reviewer without merge permission to the repository.  For me, I'd prefer the diffs minimize whitespace changes.  I'll leave it to the authoritative reviewers to decide if whitespace minimization is that important to them.  

The git plugin has a very different problem with its large number of unmerged pull requests.  Whitespace changes there can make things much harder for developers.  I don't think this plugin has that problem.

Reply:
Personally, I agree with @MarkEWaite. Mixing whitespace-only changes into a PR that changes actual content makes it harder to thoroughly review changes, so I'd appreciate if you could revert that part of the PR. 

Given this plugin is updated relatively infrequently, so conflicts are unlikely, and the changes are in docs, not code, you could open a separate PR with those changes if you wanted, but without some kind of whitespace linter as part of the PR build process, trying to enforce a line limit seems to result in back-and-forth changes between contributors with different editor settings, which we'd like to avoid.